apps/poc-state-plugin/src/plugin.ts

```diff
@@ -432,3 +438,56 @@ function createImage(data: Uint8Array, mimeType: string) {
     })
     .catch((err) => console.error(err));
 }
+
+function createMargins() {
```

Comment:
The name is confusing. The function is about creating guides, not margins. Margins are related to top/bottom/right/left padding. Does that make sense?

Reply:
This is because I want the guides to create the "margins" for a frame. I'm leaving as is for the moment.